edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/NestedDeviceScopeApiClient.cs

```diff
@@ -140,6 +140,7 @@ async Task<ScopeResult> GetIdentitiesInTargetScope(Uri uri, Option<string> conti
                 var payload = new NestedScopeRequest(this.batchSize, continuationToken.OrDefault(), authChain);
                 string token = await this.edgeHubTokenProvider.GetTokenAsync(Option.None<TimeSpan>());
                 msg.Headers.Add(HttpRequestHeader.Authorization.ToString(), token);
+                msg.Headers.Add(Constants.ServiceApiIdHeaderKey, this.actorEdgeDeviceId + "/" + this.moduleId);
```

Comment:
Why this change?

Reply:
Should we add this header only if nested edge is enabled?

Reply:
this header is ignored by IoT Hub, so it's safe to tack on.

---
In reply to: [451893193](https://github.com/Azure/iotedge/pull/3149#discussion_r451893193) [](ancestors = 451893193)